azure-pipelines-pr.yml

```diff
@@ -33,23 +33,24 @@ steps:
       npm run buildfast
     displayName: build ci
 
-  - task: AzureUpload@2
-    displayName: Upload website for PR deploy
-    inputs:
-      azureSubscription: $(azureSubscription)
-      BlobPrefix: $(deployBasePath)
-      ContainerName: '$web'
-      SourcePath: 'apps/fabric-website/dist'
-      storage: $(azureStorage)
-
-  - task: GithubPRStatus@0
-    displayName: 'Update PR deploy site github status'
-    inputs:
-      githubOwner: microsoft
-      githubRepo: fluentui
-      githubContext: 'Deployed website'
-      githubDescription: 'Click "Details" to go to the deployed website for this pull request'
-      githubTargetLink: $(deployUrl)/index.html
+  # disable in this branch
+  # - task: AzureUpload@2
+  #   displayName: Upload website for PR deploy
```

Comment:
Disabling this as this setup no longer works, if a PR deployed site is still needed for this branch, we can set it up again later